ui_framework/components/tabs/_index.scss

```diff
@@ -1,4 +1,10 @@
 $tabBackgroundColor: #FFF;
 $tabHoverBackgroundColor: #F2F2F2;
 
+// Dark theme colors
+$tabColor--darkTheme: #cecece;
+$tabBackgroundColor--darkTheme: #333333;
+$tabHoverBackgroundColor--darkTheme: #777777;
+$tabBorderColor--darkTheme: #777777;
```

Comment:
This is fantastic! I'd just suggest looking for global color variables (in https://github.com/elastic/kibana/blob/master/ui_framework/components/_variables.scss#L47), to which you can assign _these_ variables. For example:

```scss
$tabColor--darkTheme: $globalTextColor--darkTheme;
$tabBackgroundColor--darkTheme: $globalBackgroundColor--darkTheme;
// etc...
```

This way all of the components will derive the same color palette from a shared source of truth. If there are any colors which don't exist in the `_variables.scss` file, then feel free to just assign the hex value to a component-specific variable in this file.